docs/nodes/nodes-library/nodes/Dropbox/README.md

```diff
@@ -0,0 +1,48 @@
+---
+permalink: /nodes/n8n-nodes-base.dropbox
+---
+
+# Dropbox
+
+[Dropbox](https://dropbox.com) is a cloud-based file storage and sharing service, accessible through multiple devices.
+
+::: tip 🔑 Credentials
+You can find authentication information for this node [here](../../../credentials/Dropbox/README.md).
+:::
+
+## Basic Operations
+
+- File
+	- Copy a file
+	- Delete a file
+	- Download a file
+	- Move a file
+	- Upload a file
+- Folder
+	- Copy a file
+	- Create a file
+	- Delete a file
+	- List files and folders in the folder
+	- Move a folder
```

Comment:
Could you update it to match:
![image](https://user-images.githubusercontent.com/48641776/86330794-d1744c00-bc65-11ea-8550-c46808183291.png)